packages/web-worker/src/notifier.js

```diff
@@ -21,8 +22,13 @@ export const Bugsnag = {
     if (typeof opts === 'string') opts = { apiKey: opts }
     if (!opts) opts = {}
 
-    // eslint-disable-next-line no-undef
-    const internalPlugins = [pluginClientIp, pluginBrowserDevice(navigator, null), pluginPreventDiscard, pluginWindowOnError(self, 'worker onerror')]
+    const internalPlugins = [
+      pluginClientIp,
+      pluginBrowserDevice(navigator, null),
+      pluginPreventDiscard,
+      pluginWindowOnError(self, 'worker onerror'), // eslint-disable-line no-undef
+      pluginWindowUnhandledRejection(self) // eslint-disable-line no-undef
```

Comment:
I think you can tell eslint this runs in the context of a worker by adding this at the top of the file:

```js
/* eslint-env worker */
```

https://eslint.org/docs/latest/use/configure/language-options#specifying-environments

Reply:
you can! thank you :)